src/utils/dom.ts

```diff
@@ -0,0 +1,3 @@
+export const $ = <E extends Element>(selector: string, baseElement: E | Document = document): E | null => {
+  return baseElement.querySelector(selector) as E;
+};
```

Comment:
반환값이 타이핑되어 있어서 assertion은 불필요해보이는데 맞을까요~?

```suggestion
export const $ = <E extends Element>(selector: string, baseElement: E | Document = document): E | null => {
  return baseElement.querySelector(selector);
};
```